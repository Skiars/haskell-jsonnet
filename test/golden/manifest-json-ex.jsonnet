std.manifestJsonEx(
  {
      x: [1, 2, 3, true, false, null,
          @"string\n""string""\n"],
      y: { a: 1, b: 2, c: [1, 2] }
  },
  "  "
)
