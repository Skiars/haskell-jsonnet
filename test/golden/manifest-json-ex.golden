"{\n  \"x\": [\n    1,\n    2,\n    3,\n    true,\n    false,\n    null,\n    \"string\\\\n\\\"string\\\"\\\\n\"\n  ],\n  \"y\": {\n    \"a\": 1,\n    \"b\": 2,\n    \"c\": [\n      1,\n      2\n    ]\n  }\n}"